### ISSUE TITLE
Use tf_frame_prefix in firmware_message_converter

### ISSUE THREAD

**Issue by bjsowa** [LeoRover/leo_robot#3]
The `tf_frame_prefix` argument in `leo_bringup.launch` file
https://github.com/LeoRover/leo_robot/blob/af90ed40107fa862c234575a4b8033da0622a7fd/leo_bringup/launch/leo_bringup.launch#L4
should affect the `frame_id` fields of the messages published by the `firmware_message_converter` node.

For example, when `leo_bringup.launch` is included with the `tf_frame_prefix` argument set to non-empty string:
```xml
<include file="$(find leo_bringup)/launch/leo_bringup.launch">
    <arg name="tf_frame_prefix" value="robot1/" />
</include>
```
the example message on the `/wheel_odom_with_covariance` topic should look like this:
```yaml
header: 
  seq: 17020
  stamp: 
    secs: 1641572792
    nsecs: 348135658
  frame_id: "robot1/odom"
child_frame_id: "robot1/base_link"
pose: 
  pose: 
    position: 
      x: 0.0
      y: 0.0
      z: 0.0
    orientation: 
      x: 0.0
      y: 0.0
      z: 0.0
      w: 1.0
  covariance: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
twist: 
  twist: 
    linear: 
      x: 0.0
      y: 0.0
      z: 0.0
    angular: 
      x: 0.0
      y: 0.0
      z: 0.0
  covariance: [0.0001, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.001]
```

**Comment by Bitterisland6** [comment]
Will the argument value always be finished with "/" like in the example ("robot1/"), or should i check whether or not there is slash at the end?

**Comment by bjsowa** [comment]
The argument value does not have to be finished with "/" but you should not treat this character any different. The `tf_frame_prefix` should not work like namespaces in ROS [Graph Resource Names](http://wiki.ros.org/Names#Graph_Resource_Names). Instead it should just add the prefix value to the published frame ids.

For example, setting the `tf_frame_prefix` to `robot1` should result in frame ids like `robot1base_link`, `robot1odom` etc.

**Comment by bjsowa** [comment]
Resolved by #4
